sailor/pai/utils.py

```diff
@@ -0,0 +1,63 @@
+"""Module for various utility functions, in particular those related to fetching data from remote oauth endpoints."""
+
+import logging
+import warnings
+
+from ..utils.oauth_wrapper import get_oauth_client
+from ..utils.config import SailorConfig
+from ..utils.utils import DataNotFoundWarning
+from ..assetcentral.utils import _compose_queries
+
+LOG = logging.getLogger(__name__)
+LOG.addHandler(logging.NullHandler())
+
+
+def _fetch_data(endpoint_url, unbreakable_filters=(), breakable_filters=()):
```

Comment:
do we really need a separate `_fetch_data` here? What are the differences to the version in accetcentral?

Reply:
you found it already why i left it def _fetch_data(endpoint_url, unbreakable_filters=(), breakable_filters=()):
    """Retrieve data from the pai service."""
    filters = _compose_queries(unbreakable_filters, breakable_filters)
    oauth_client = get_oauth_client('pai')

Reply:
I'm okay with this now. We can refactor later. (I'm adding this to our list )

Reply:
fetch data is removed. using one from asset central with a parameter. default value asset_central